netstandard/ref/System.IO.Compression.cs

```diff
@@ -4,6 +4,47 @@
 
 namespace System.IO.Compression
 {
+    public partial struct BrotliDecoder : System.IDisposable
```

Comment:
I assume there's some reason this and BrotliEncoder are structs? It seems a little odd, but obviously if that's the way they are in Core, they can't change for Standard. (And I'm sure there *is* a good reason, of course.)

Reply:
See [below](https://github.com/dotnet/standard/pull/923#discussion_r227103913).